### DISCUSSION TITLE
Upload image button not shown on confirmation page

### DISCUSSION THREAD

**Discussion by atakansaracoglu** [community/community#17763] [GitHub Education]
Hi guys, I'm trying to apply for github student pack but i couldnt pass the verification process because of my poor webcam.
In help section, it says "there is an upload image button". but i couldnt see it. 

Do you have any ideas?

Thanks in advance.

**Comment by shaikmomin** [comment]
Try refreshing the page or probably "drag and drop" your png or jpg file 😁

**Reply by TungusSs** [reply]
Nothing helps. In YouTube videos, most have a photo upload button, but this guy and I don't have them. Support responds in a template not at all on the topic of the question, but no one pays special attention to such discussions either, and unfortunately there is no one to expect normal help from.

**Reply by datsfilipe** [reply]
Well, the drag and drop thing worked for me. Thanks!

**Comment by TungusSs** [comment]
Hey man! I have the same problem too...

**Reply by CodeMarshal007** [reply]
I also have the same problem, I've used all my devices but still can't see an upload button.

**Comment by atakansaracoglu** [comment]
I have refreshed many times, tried to drag and drop. Nothing helps.

**Comment by atakansaracoglu** [comment]
Now chatting with support, and they are sending TEMPLATE messages like robot.

**Reply by TungusSs** [reply]
I saw that they give a link to the answer to a user who is faced with about the same problem as ours, so that he "does not despair and applies further, because at one point someone from the Github team will notice your activity and you will eventually approve the application"

**Comment by mlarasusan** [comment]
Sorry about that. I have a few recommendations for you too here: 

:point_right: Check [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) to learn how to get the GitHub Student Developer Pack without a student ID
:point_right: Also, check the reviews of these two students: [here](https://github.com/github-community/community/discussions/31#discussioncomment-2561585) & [here](https://github.com/orgs/github-community/discussions/16)
:point_right: And here is the [link](https://support.github.com/contact/education) to contact our support team [✨](https://emojipedia.org/sparkles/)
:point_right: [FAQs](https://github.com/orgs/github-community/discussions/9)

Let me know if it helps :]

**Comment by proarafat** [comment]
If you have more problems, you can talk to<a href="https://support.github.com/"> GitHub Support<a>
They will fix these bugs for you. So for those of you who are having this kind of problem, talk to <a href="https://support.github.com/"> GitHub Support<a>
☺☺☺☺

**Comment by elisemoe** [comment]
Hi @atakansaracoglu 👋  sorry for the frustration! Would you be able to apply from a phone camera?

Sometimes, for a number of reasons, our system needs additional levels of verification and asks that you take an image rather than upload one. Other suggestions might be to try applying while you are on campus, if able to.

**Reply by Jade-Jones-dev** [reply]
I tried for two days to take pictures of different proof. Each time I took a picture it said that I needed to use something else. I have contacted support and not had a response back

**Comment by yang-dongxu** [comment]
Hi, I have met the same problem today, and I find the "upload" button appears when another email was selected. 
Hope it helps you.

**Reply by adriano-ramirez** [reply]
I have the same problem and your tip didn't have worked for me :(

**Comment by DavincyProject** [comment]
i have same problem too :(

**Reply by albatros68** [reply]
Excuse me, can you check if this site works? I have several loading problems I wrote to the administrators and they told me that they don't have any problems. The site is as follows: https://fr1.badoo.com/encounters/
Thank you in advance

**Comment by dragao1995** [comment]
The problem here is that my college is distance education (EAD), so I can't go to the campus, and the upload button has also disappeared, and when I try to send the ID, it gets denied.
